charts/netdata/templates/psp.yaml

```
@@ -9,8 +9,8 @@ metadata:
     chart: {{ .Chart.Name }}-{{ .Chart.Version }}
 spec:
   allowPrivilegeEscalation: true
-  allowedCapabilities:
-  - '*'
+  defaultAddCapabilities:
+    - SYS_PTRACE
```

```suggestion
  allowedCapabilities:
    - SYS_PTRACE
    - CAP_SYS_ADMIN
```

Those are needed by [apps.plugin](https://github.com/netdata/netdata/tree/master/collectors/apps.plugin#appsplugin) and [cgroups.plugin](https://github.com/netdata/netdata/tree/master/collectors/cgroups.plugin#cgroupsplugin).

See https://github.com/netdata/helmchart/blob/915ea3bd202aef27385638dc73fc61071cecd40b/charts/netdata/templates/daemonset.yaml#L157-L161

And I think that 'allowedCapabilities' is better (more secure) than 'allowedCapabilities'.